yotierror/response.go

```diff
@@ -65,8 +65,8 @@ func formatResponseMessage(response *http.Response, httpErrorMessages ...map[int
 		return defaultMessage
 	}
 
-	body, _ := io.ReadAll(response.Body)
-	response.Body = io.NopCloser(bytes.NewBuffer(body))
+	body, _ := ioutil.ReadAll(response.Body)
```

Comment:
`ioutil` is deprecated, `io` was fine, these errors should've really been handled before though..